torch/optim/lr_scheduler.py

```diff
@@ -700,6 +700,76 @@ def load_state_dict(self, state_dict):
             self._schedulers[idx].load_state_dict(s)
 
 
+class PolynomialLR(_LRScheduler):
+    """Decays the learning rate of each parameter group using a polynomial function,
+    to reach a min_lr in the given total_iters. When last_epoch=-1, sets initial lr as lr.
+
+
+    Args:
+        optimizer (Optimizer): Wrapped optimizer.
+        total_iters (int): After this step, learning rate stops decreasing.
+        min_lr (float or list): A scalar or a list of scalars. A
+            lower bound on the learning rate of all param groups
+            or each group respectively. Default: 0.
+        power (int): The power of the polynomial. Defaults to linear, 1.0.
+        verbose (bool): If ``True``, prints a message to stdout for
+            each update. Default: ``False``.
+
+    Example:
+        >>> # Assuming optimizer uses lr = 0.001 for all groups
+        >>> # lr = 0.0010   if epoch == 0
+        >>> # lr = 0.0008   if epoch == 1
+        >>> # lr = 0.0006   if epoch == 2
+        >>> # lr = 0.0004   if epoch == 3
+        >>> # lr = 0.0      if epoch >= 4
```

Comment:
Are sure these values are right? I believe for the proposed configuration it would be more like:
```
0 0.001
1 0.00075
2 0.0005
3 0.00025
```

Reply:
I checked the values. You're right. Corrected.